code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -92,62 +108,68 @@
 	var/emitteroverloadcd = 100
 	var/emittersemicd = FALSE
 
-	var/overload_ventcrawl = 0
-	var/overload_bulletblock = 0 //Why is this a good idea?
-	var/overload_maxhealth = 0
-	var/silent = FALSE
-	var/brightness_power = 5
+	/// Bool that determines if the pAI can refresh medical/security records.
+	var/refresh_spam = FALSE
+	/// Cached list for medical records to send as static data
+	var/list/medical_records = list()
+	/// Cached list for security records to send as static data
+	var/list/security_records = list()
+
+	var/list/available_software = list(
+		"crew manifest" = 5,
+		"digital messenger" = 5,
+		"atmosphere sensor" = 5,
+		"photography module" = 5,
+		"camera zoom" = 10,
+		"printer module" = 10,
+		"remote signaler" = 10,
+		"medical records" = 10,
+		"security records" = 10,
+		"host scan" = 10,
+		"medical HUD" = 20,
+		"security HUD" = 20,
+		"loudness booster" = 20,
+		"newscaster" = 20,
+		"door jack" = 25,
+		"encryption keys" = 25,
+		"internal gps" = 35,
+		"universal translator" = 35,
+	)
 
 /mob/living/silicon/pai/add_sensors() //pAIs have to buy their HUDs
 	return
 
-/mob/living/silicon/pai/handle_atom_del(atom/A)
-	if(A == hacking_cable)
+/mob/living/silicon/pai/handle_atom_del(atom/deleting_atom)
```

Comment:
bike shedding (feel free to ignore)
`deleting_atom` feels like a strange var name, `atom_to_del` or something similar would be better imo

Reply:
I took it because thats what parent uses